shared/src/license.ts

```diff
@@ -39,13 +33,54 @@ export type LicenseStatus =
   | "SUSPENDED"
   | "REVOKED";
 
-export interface LicenseData {
+export type LicenseDetails = {
   nameAndAddress: LicenseSearchNameAndAddress;
-  completedSearch: boolean;
-  expirationISO?: string;
+  licenseStatus: LicenseStatus;
+  expirationDateISO: string | undefined;
+  lastUpdatedISO: string;
+  checklistItems: LicenseStatusItem[];
+  hasError?: boolean;
+};
+
+export const enabledLicensesSources = {
+  "Employment & Personnel Service-Career Counseling Service": "RGB",
+  "Employment & Personnel Service-Employment Agency": "RGB",
+  "Employment & Personnel Service-Entertainment/Booking Agency": "RGB",
+  "Employment & Personnel Service-Nurses Registry": "RGB",
+  "Employment & Personnel Service-Resume Service": "RGB",
+  "Health Care Services": "RGB",
+  "Health Club Services": "RGB",
+  Telemarketers: "RGB",
+  "Ticket Brokers": "RGB",
+};
+
+export const taskIdLicenseNameMapping = {
```

Comment:
What if we create a list that maps license calendar events to the LicenseName and do the same for reinstatements. If we go this route, we should be able to remove the need to add the LicenseName attribute in the individual files.

Reply:
leaving alone